docker-gen.go

```diff
@@ -58,6 +58,13 @@ type Volume struct {
 	ReadWrite bool
 }
 
+type Mount struct {
+	Source      string
+	Destination string
+	Mode        string
+	RW          bool
```

Comment:
On Wed, Sep 09, 2015 at 11:18:56AM -0700, Jason Wilder wrote:
>    In docker-gen.go:
>
>  > @@ -58,6 +58,13 @@ type Volume struct {
>  >    ReadWrite bool
>  >  }
>  >
>  > +type Mount struct {
>  > +  Source      string
>  > +  Destination string
>  > +  Mode        string
>  > +  RW          bool
>
>    Can you rename this to ReadWrite to be consistent w/ Volume?

Since 'ReadWrite' is synthesized from 'VolumesRW' in APIs < 1.20, doesn't it
sense to maintain consistency w/the API and dockerclient in this case, where
the 'Mount' struct is a verbatim copy from go-dockerclient
(https://github.com/fsouza/go-dockerclient/blob/master/container.go#L224.)

Granted i have deviated from the underlying library by turning the Mounts
array into a map, but i feel this makes it much more useful in the template
use-case (FWIW, i feel the API implementation of the mounts as an array
rather than a map is a regression from the previous version).

rick